### PR TITLE
Enhancement/revamp search crw 138

### PR DESCRIPTION
**Pull request by AbdallaEmadEldinAbdalla**
[CRW-138](https://techxarya.atlassian.net/browse/CRW-138)
[CRW-139](https://techxarya.atlassian.net/browse/CRW-139)

[CRW-138]: https://techxarya.atlassian.net/browse/CRW-138?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CRW-139]: https://techxarya.atlassian.net/browse/CRW-139?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/shared/assetSearchInputWithDropdown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+                    {keyword !== '' && <Button className="p-1.5 bg-black-1 rounded-xl absolute right-4" onClick={() => {
+                        setKeyword('');
+                        setFocused(false);
+                    }}>
+                        <CloseIcon className="stroke-current text-[#89939F] w-2 h-2" />
+                    </Button>}
+                </Row>
+            </Button>
+            {focused && <Col className={clsx("w-[400px] h-[300px] bg-grey-2 top-16 right-0 absolute items-center rounded-md overflow-scroll p-4", styles.list)}>
```
Comment (alex-greenbull) [inline src/components/shared/assetSearchInputWithDropdown/index.tsx:83]: change h-[300px] to max-h-[300px]
change to overflow-scroll to overflow-auto

goal is to have no scroll bar if no keyword
Screenshot__001284

File: src/components/shared/assetSearchInputWithDropdown/index.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+}
+
+export const SearchAssetInput = ({ onClick, t }: AssetDropdownTypes) => {
+    const [coins, setCoins] = useState<AssetType[]>([]);
+    const [loading, setLoading] = useState<boolean>(true);
+    const [keyword, setKeyword] = useState<string>('');
+    const [focused, setFocused] = useState<boolean>();
+    const { push } = useRouter();
+
+    const resultLimit = useMemo(() => keyword !== '' ? 50 : 10, [keyword]);
```
Comment (alex-greenbull) [inline src/components/shared/assetSearchInputWithDropdown/index.tsx:29]: change 10 to 5

File: src/components/shared/assetSearchInputWithDropdown/index.tsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                        setFocused(false);
+                    }}>
+                        <CloseIcon className="stroke-current text-[#89939F] w-2 h-2" />
+                    </Button>}
+                </Row>
+            </Button>
+            {focused && <Col className={clsx("w-[400px] h-[300px] bg-grey-2 top-16 right-0 absolute items-center rounded-md overflow-scroll p-4", styles.list)}>
+                {loading ? <LoadingSpinner /> :
+                    coins.length === 0 ?
+                        <span className="w-full mx-4 text-center">No asset with this keyword!<br /><br />{keyword}</span>
```
Comment (alex-greenbull) [inline src/components/shared/assetSearchInputWithDropdown/index.tsx:86]: Translation

File: src/components/svg/searchIcon.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 export const SearchIcon = () => (
     <svg width="42" height="43" viewBox="0 0 42 43" fill="none" xmlns="http://www.w3.org/2000/svg">
-        <rect width="42" height="43" rx="6" fill="#213660" />
+        <rect width="42" height="43" rx="6" fill="transparent" />
         <path d="M19.6531 25.2919C22.5867 25.2919 24.9648 22.9151 24.9648 19.9833C24.9648 17.0514 22.5867 14.6746 19.6531 14.6746C16.7196 14.6746 14.3415 17.0514 14.3415 19.9833C14.3415 22.9151 16.7196 25.2919 19.6531 25.2919Z" stroke="#558AF2" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round" />
         <path d="M23.4471 23.7751L28 28.3254" stroke="#558AF2" stroke-width="2.5" stroke-linecap="round" stroke-linejoin="round" />
     </svg>
```
Comment (ameer-greenbull) [inline src/components/svg/searchIcon.tsx:6]: This icon is duplicate use the one used in AssetSelector or remove both and replace with heroIcon

**Review by ameer-greenbull — CHANGES_REQUESTED** [review]

**Review by ameer-greenbull — APPROVED** [review]

**Pull request merged**